aiogram_dialog/window.py

```diff
@@ -73,11 +75,14 @@ async def process_callback(self, c: CallbackQuery, dialog: Dialog,
     async def render(self, dialog: Dialog, manager: DialogManager) -> NewMessage:
         logger.debug("Show window: %s", self)
         current_data = await self.load_data(dialog, manager)
+        reply_markup = await self.render_kbd(current_data, manager)
+
         return NewMessage(
             chat=get_chat(manager.event),
             text=await self.render_text(current_data, manager),
-            reply_markup=await self.render_kbd(current_data, manager),
+            reply_markup=reply_markup,
             parse_mode=self.parse_mode,
+            show_mode=ShowMode.SEND if isinstance(reply_markup, ForceReplyMarkup) else ShowMode.AUTO,
```

Comment:
`AUTO` means that we will detect automatically if we need to send or edit message. Like in case of switching from text to media. We do not need this logic here